builds/otel.sh

```diff
@@ -14,11 +14,18 @@
 # limitations under the License.
 
 set -x -e
-DESTDIR=$1
+
+ROOTDIR="$1"
 otel_dir=/opt/google-cloud-ops-agent/subagents/opentelemetry-collector
-DESTDIR="${DESTDIR}${otel_dir}"
+DESTDIR="${ROOTDIR}${otel_dir}"
+
+# Default values for go build arguments.
+BINARYLOCATION="$ROOTDIR/otelopscol"
+LDFLAGS=""
```

Comment:
When I had `-ldflags` with the string as well there were weird failures because of the quote characters and I couldn't get it working, so thought it would be harmless to do it this way instead. It does function fine without `-ldflags` as `-ldflags` is empty by default.